apps/web/src/server/applications/case/edit/case/applications-edit-case.controller.js

```diff
@@ -536,3 +548,47 @@ export async function updateApplicationsEditCaseZoomLevel(request, response) {
 			: `/applications-service/case/${updatedCaseId}/project-information`
 	);
 }
+
+/**
+ * View the form step for editing is material change
+ *
+ *
+ * @type {import('@pins/express').RenderHandler<ApplicationsCreateCaseIsMaterialChangeRes,
+ * {}, {}, {}, {}>}
+ */
+export async function viewApplicationsEditIsMaterialChange(request, response) {
+	if (!featureFlagClient.isFeatureActive('applics-156-material-changes')) {
+		const referrerURL = request.header('Referrer') || '/';
```

Comment:
I don't think this is needed, express should handle this itself. Unless there was a reason you needed to add it.

Reply:
Ignore this, I misread the code 👍